### DISCUSSION TITLE
What Linux Distros do Ventoy support persistent storage installations?

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1238] [Q&A]
What Linux Distros do Ventoy support persistent storage installations? Does it support most Debian, Ubuntu and Arch based distros?

**Comment by ventoy** [comment]
What do you mean about persistent installation? Install the linux to local disk? Or run LiveCD with persistence support?
Do you mean this: https://www.ventoy.net/en/plugin_persistence.html

**Reply by trymeouteh** [reply]
By having the distro installed on a USB stick or harddrive and be able to boot up the distro, and do work on it like install apps, create files, etc and when I boot the USB stick or drive down, the changes made will still be remained the next time I boot it up. Essentially the ability to make a portable OS on a stick.

**Reply by ventoy** [reply]
I don't understand what you mean?  Linux can be installed on Local Disk and can also be installed on USB. There is no difference bwtween them.

**Reply by tazihad** [reply]
@ventoy I think he meant Using ventoy to install linux as external hard drive. When if he plugin usb again he should see his previous work saved.

**Reply by duven87** [reply]
yeah... I was wondering about that too, I find it great to have all the isos in ventoy but I would also like to use one of them in live and be able to save the changes I make to the system, package installation, updates, configurations... is that possible ?

**Reply by trymeouteh** [reply]
I would like to know how to have a live usb stick like tails but for other linux distros.
